QModManager/Checks/NitroxCheck.cs

```diff
@@ -1,4 +1,4 @@
-﻿using Harmony;
+﻿using HarmonyLib;
```

Comment:
Just a bit curious: which namespace will new mods need to use?

Reply:
Either. HarmonyLib if they're targeting Harmony2/HarmonyX. Harmony if they're still targeting old Harmony 1.x. The shimmer takes care of conversion from Harmony 1 namespace references to Harmony 2 namespace references.

Reply:
Ie, they use the namespace that's in the Harmony dll they are referencing as normal.

Reply:
> Either. HarmonyLib if they're targeting Harmony2/HarmonyX. Harmony if they're still targeting old Harmony 1.x. The shimmer takes care of conversion from Harmony 1 namespace references to Harmony 2 namespace references.

This should be added somewhere, like in a wiki or something.

Reply:
Yeah, my plan is to look at updating docs when we're actually looking to update :)